api/client/crop_model.py

```diff
@@ -11,16 +11,16 @@ class CropModel(GroClient):
 
     def compute_weights(self, crop_name, metric_name, regions):
         """Compute a vector of 'weights' that can be used for crop-weighted
-        average across regions.
+        average across regions, as in :meth:`~.compute_crop_weighted_series`.
```

Comment:
I just added these `:meth:~` links like in https://github.com/gro-intelligence/api-client/pull/141/files#diff-9adf8c29ac9359811cbf476c22bab58bR169 which make them clickable in the docs